aviatrix/resource_aviatrix_account.go

```diff
@@ -853,6 +864,9 @@ func resourceAviatrixAccountRead(ctx context.Context, d *schema.ResourceData, me
 			d.Set("awss_cap_cert_key_path", acc.AwsSCapCertKeyPath)
 			d.Set("aws_ca_cert_path", acc.AwsCaCertPath)
 		}
+		if acc.GroupName != "" {
```

Comment:
1. It is possible to change the RBAC groups attached to the account using the other RBAC group resources, which will cause this attribute to show a diff. Since this is a create only attribute, we need a way to prevent this diff. It should either always have a diff suppress func or should be marked computed some how.
2. Since the attribute is a list of strings, it should be converted to a slice of strings to make sure it's ok. 
3. This if check will have some errors. Lets say that after the account is created with associated RBAC groups, the API returns the empty string (this could be because the API didn't attach the groups on create but returned successfully or because a user removed the access group on the UI). The attribute will not be set and there will be no diff even though there was a change. We can just remove the if condition.

Reply:
add DiffSuppressFunc to prevent each update will no change.